.github/workflows/resource-indexer.yml

```diff
@@ -0,0 +1,39 @@
+name: rebuild the resource collection index
+
+on:
+  # Run at ~4am UTC time which is (± an hour) 4am UK, 5am Switzerland, midnight
+  # US east coast, 9pm US west coast so that for most users (and most
+  # developers) the index regenerates overnight
+  schedule:
+    - cron: '* 4 * * *'
```

Comment:
Noticed this as I was quickly looking: this crontab spec is "every minute of the 04 hour". You want `0 4 * * *` (or similar, replacing 0 with 0–59).

Reply:
Thanks - fixed up 👍